VERSION

```diff
@@ -1,4 +1,4 @@
 VERSION_MAJOR=2
 VERSION_MINOR=4
 VERSION_PATCH=2
-VERSION_SUFFIX=-dev
+VERSION_SUFFIX=
```

Comment:
The base of this branch ain't right: it seems to be based on the tag. So, will need to wait for 2.4.2 to be released, then merged (@gdemonet), then reparent this change on `development/2.4`.

Reply:
Reparented on `development/2.5` after 2.5.1 tag was merged.